collective/ogtags/browser/viewlets.py

```diff
@@ -1,12 +1,12 @@
+from zope.component import (ComponentLookupError, getMultiAdapter, getUtility,
+                            queryMultiAdapter)
```

Comment:
I have seen this `import` change before in a previous pull request. Do you have some tool like `isort` that automatically does this? I prefer the way it is on master, with each import on one line, which is according to the Plone style guide. See http://docs.plone.org/develop/styleguide/python.html#about-imports

I don't mind much in a pull request, but I will feel free to redo the import sorting after merge. :-)